Test

Edmaruu opened the pull request:
Test du solver avec la fonction de recuit

ButtePaillade91 committed:
Résolution sudoku marche

Edmaruu committed:
test solver

Edmaruu committed:
Merge branch 'main' into test

jsboigeEPF commented:
Désolé mais je n'en peux plus de voir tout ce travail livré au mauvais endroit. Laissez le projet backtracking tranquille, renommez vos projets initiaux s'ils n'étaient pas correctement nommés et travaillez dedans, pas à côté !! Pour l'heure je ne fusionne pas. Mettez-moi un commentaire quand votre PR est propre ou si vous rencontrez des difficultés. Je vous fais une PR pour que vous voyez comment les autres ont travaillé.

Edmaruu committed:
Solver par recuit

le solver par recuit est correctement installé par rapport à la méthodologie demandé

Edmaruu committed:
ajout de la grille du benchmark pour résoudre avec notre fonction de côut
